src/anemoi/training/diagnostics/mlflow/auth.py

```diff
@@ -98,7 +98,7 @@ def login(self, force_credentials: bool = False, **kwargs: dict) -> None:
         Parameters
```

Comment:
Minor suggestion, but I'd add a comment to the login docstring mentioning that the validity of the refresh token is 30 days. The command provides that information once executed, but I feel like in this piece of code there is no mention about that, so it could be helpful from a developer perspective to know about it

Reply:
done!